contrib/node/README.md

```diff
@@ -1,53 +1,81 @@
-# Plugin to support JS/Node
+# Plugin to support JavaScript and Node.js
 
-This is a plugin to integrate Node/NPM with Pants.
+This is a plugin to integrate Node.js with Pants.
 
 # Targets
 
-You can specify node_remote_module targets referring to remote 3rd party Node modules, and
-you can specify node_module targets defining projects in the source tree.
+Node.js targets in pants are designed to work with existing Node.js package management tools. A node_module target will work with a package.json
+source file in the same directory, combining the BUILD dependencies with the additional fields from the source package.json to form the complete
+package.json used for tasks on the target.
 
-node_module targets can depend on node_remote_module targets in the "dependencies" field of
-their specification in a BUILD file. A node_module target will work with a package.json
-source file in the same directory, combining the BUILD dependencies with the additional fields from
-the source package.json to form the complete package.json used for tasks on the target.
 
-The way this currently works is, when resolving a target, Pants drops the dependencies,
-devDependencies, peerDependencies and optionalDependencies fields from the target's corresponding
-package.json file, and installs with a dependencies field populated from the BUILD file target
-dependencies. This has the benefit of letting Pants work with an existing package.json file
-for a project that can be installed via "npm install" without Pants.
-However a project that takes advantage of this ability to be installed via a normal "npm install"
-in the source tree would need its dependencies duplicated in the BUILD target and package.json file.
-Obviously this is not desirable. Over time, Pants support for NPM should improve to work with a
-fully functional package.json with no duplication of information in the BUILD file.
+`node_module`: A node module target.
```

Comment:
Markdown will not render these as a list unless you prefix them with either bullets or numbers (they get mashed into a single paragraph).

Reply:
If you preview the docsite, or use the github-builtin markdown viewer (https://github.com/pantsbuild/pants/blob/981d66ff4114d8a1f2da74423db35ef936355a7d/contrib/node/README.md), you can check out how this render.